packages/auth/docs/stories/emailPasswordSignUpForm.story.js

```diff
@@ -0,0 +1,183 @@
+                                                    fill="#01B6F5"
+                                                    points="0.692628832 10.6429587 17.1665904 10.641762 10.5923442 5.3266476 7.35346682 5.3266476"
+                                                />
+                                                <path
+                                                    d="M1.56660706,0.0114988558 C1.33652511,0.0114988558 1.12691431,0.0927116705 0.969221968,0.224107551 L8.35941747,7.2328833 C8.69481754,7.55093822 9.24358842,7.55093822 9.57898849,7.2328833 L16.969222,0.224073227 C16.8115296,0.0926773455 16.6019188,0.0114988558 16.3718369,0.0114988558 L1.56660706,0.0114988558 L1.56660706,0.0114988558 Z"
+                                                    id="Shape"
+                                                    fill="#38CCFF"
+                                                />
+                                            </g>
+                                        </g>
+                                    </g>
+                                </g>
+                            </g>
+                        </g>
+                    </g>
+                </g>
+            </g>
+        </svg>
+    );
+
+    const passwordIcon = (
+        <svg xmlns="http://www.w3.org/2000/svg" width="14" height="16" viewBox="0 0 14 16">
+            <g fill="none">
+                <path
+                    fill="#009ACF"
+                    d="M6.22 0C3.605.003 1.485 1.82 1.482 4.063v2.5c0 .172.163.312.365.312h1.458c.201 0 .365-.14.365-.313v-2.5c0-1.208 1.142-2.187 2.552-2.187 1.41 0 2.552.98 2.552 2.188v2.5c0 .172.163.312.364.312h1.459c.201 0 .364-.14.364-.313v-2.5C10.957 1.82 8.837.003 6.22 0z"
+                    transform="translate(.75 .5)"
+                />
+                <path
+                    fill="#01B6F5"
+                    d="M1.812 6.25h8.587c.988 0 1.789.7 1.789 1.562v5.625c0 .863-.801 1.563-1.79 1.563H1.813c-.988 0-1.79-.7-1.79-1.562V7.813c0-.863.802-1.563 1.79-1.563z"
+                    transform="translate(.75 .5)"
+                />
+                <path
+                    fill="#FFF"
+                    d="M7.781 9.688c.004-.863-.727-1.566-1.633-1.57-.907-.003-1.644.693-1.648 1.556-.003.597.352 1.143.913 1.407L5.16 12.77c-.026.17.099.329.278.353l.046.003h1.313c.181.002.33-.137.331-.31 0-.015 0-.031-.003-.047l-.253-1.687c.554-.265.906-.803.91-1.393z"
+                    transform="translate(.75 .5)"
+                />
+            </g>
+        </svg>
+    );
+
+    return (
+        <div>
+            <div className="rainbow-p-around_large" style={logoStyle}>
+                {rainbowLogo}
+                <h3 style={logoTextStyle}>rainbow-modules</h3>
+            </div>
+            <Card className="rainbow-p-around_large" style={style}>
+                <EmailPasswordSignUpForm
+                    nameIcon={nameIcon}
+                    emailIcon={emailIcon}
+                    passwordIcon={passwordIcon}
```

Comment:
since we don't define any props the icons should be part of the component for now